.buildkite/commands/run-ui-tests.sh

```diff
@@ -27,4 +27,12 @@ install_cocoapods
 echo "--- 🧪 Testing"
 xcrun simctl list >> /dev/null
 rake mocks &
+set +e
 bundle exec fastlane test_without_building name:"$TEST_NAME" try_count:3 device:"$DEVICE" ios_version:"$IOS_VERSION"
+TESTS_EXIT_STATUS=$?
+set -e
+
+echo "--- 📦 Zipping test results"
+cd build/results/ && zip -rq WordPress.xcresult.zip WordPress.xcresult
+
+exit $TESTS_EXIT_STATUS
```

Comment:
Looking at the logs I now see that the `Error: The command exited with status 1` message that gets printed in the log due to that might misleadingly make us think that this is the "Zipping test results" steps that has failed, as opposed to the tests, so probably worth adding another header before this to separate that `exit` and clarify in logs:

```suggestion
echo "--- 🚦 Report Tests Exit Status
exit $TESTS_EXIT_STATUS

```